<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OCR GUI</string>
  </property>
  <widget class="QPushButton" name="btnQuit">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Выход</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="showd">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>170</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Button">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>239</red>
         <green>235</green>
         <blue>231</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="text">
    <string>Открыть...</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>631</width>
     <height>27</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="mylabel">
      <property name="text">
       <string>Empty</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="textOut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByKeyboard|Qt::LinksAccessibleByMouse|Qt::TextBrowserInteraction|Qt::TextEditable|Qt::TextEditorInteraction|Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConv">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>40</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Распознать</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mylabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>521</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>70</y>
     <width>105</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Сохранить...</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="mylabel_3">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>150</y>
     <width>101</width>
     <height>81</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Это не замена FineReader - это GUI для tesseract.</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <widget class="QLabel" name="mylabel_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>250</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Delvin LabDesign®</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="textInteractionFlags">
    <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>btnQuit</zorder>
  <zorder>showd</zorder>
  <zorder>textOut</zorder>
  <zorder>btnConv</zorder>
  <zorder>mylabel_2</zorder>
  <zorder>btnSave</zorder>
  <zorder>mylabel_3</zorder>
  <zorder>mylabel_4</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
